testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -877,6 +877,51 @@ def forward(self, x, kernel, bias):
 model = ConvBias()
 save_data_and_model_multy_inputs("conv_variable_wb", model, x, kernel, bias)
 
+<<<<<<< HEAD
```

Comment:
Please remove this line

Reply:
Sorry, my development tools produce these strange bugs (not shown in IDE)